### ISSUE TITLE
Is this repo still maintained?

### ISSUE THREAD

**Issue by dodysw2** [gocraft/work#120]
Planning to adopt this for a large code base, but worried looking at the number of unresolved issues and last commit date.

**Comment by matt-harvey** [comment]
I was wondering the same thing. There are open PRs that would resolve some of those issues, but those too appear to be going unattended.

**Comment by steven-zou** [comment]
Same question

**Comment by phamdt** [comment]
perhaps the maintainers would like to vet a community supporter or two to help with merging PRs and reviewing issues?

**Comment by taylorchu** [comment]
Many people wonder why there is no update for a while, and here is why: we have been working on the [v2 rewrite](https://github.com/taylorchu/work/) about a year ago. It fixes many issues (please see the readme), but most importantly, queue reliability. There were some incidents that were caused by gocraft v1, which moved jobs around in redis without atomic operation. 

v2 runs smoothly for a year now in production, so I think it is ready for you to try.

**Comment by steven-zou** [comment]
@taylorchu @cypriss @hoffoo @snorecone 

As we have built our own distributed job service based on this version, my question is, will this repo continue to accept some fix PRs?

**Comment by taylorchu** [comment]
yes. We are also looking for a maintainer for v1. If someone wants to be one, please comment here.

**Comment by emq** [comment]
> v2 runs smoothly for a year now in production, so I think it is ready for you to try.

Oh my, why this gem is so deeply hidden? 🙈 maybe v2 is worth mentioning in the readme here for better visibility? 👏

**Comment by digininja** [comment]
Will the documentation for V1 still apply to V2? I'm fairly new to Go so the more documentation the better, obviously V1 has a lot, V2 doesn't seem to have any yet.

**Comment by hibiken** [comment]
If anyone is interested in trying out an alternative library, please check out [Asynq](github.com/hibiken/asynq). 
It's currently under active development. 

Any feedback would be greatly appreciated! Thank you!

**Comment by mmacheerpuppy** [comment]
I'd maintain this!

**Comment by mmacheerpuppy** [comment]
@hibiken

**Comment by aanari** [comment]
Me too!

**Comment by sdabhi23** [comment]
@taylorchu would be happy to help with the maintenance of the v1 repo. Also, you should consider pinning this issue
